src/conda_constructor/uninstall.py

```diff
@@ -217,10 +210,17 @@ def _remove_environments(prefix: Path, prefixes: list[Path]):
         if conda_root_prefix and conda_root_prefix == env_prefix:
             del os.environ["CONDA_ROOT_PREFIX"]
             reset_context()
+        # Conda does not remove the default environment, so set it to something else temporarily
+        if default_activation_prefix == env_prefix:
```

Comment:
Should probably compare the `.resolve()`d paths in case there are symlinks or something like that.